src/utils/components/PendingChanges/ModalPendingChangesAlert/ModalPendingChangesAlert.tsx

```diff
@@ -1,22 +1,27 @@
-import React from 'react';
+import React, { FC } from 'react';
 
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 
 import { PendingChangesAlert } from '../PendingChangesAlert/PendingChangesAlert';
 
-type ModalPendingChangesAlertProps = {
+export type ModalPendingChangesAlertProps = {
   isChanged: boolean;
+  showMigrateOption?: boolean;
 };
 
-export const ModalPendingChangesAlert: React.FC<ModalPendingChangesAlertProps> = ({
+export const ModalPendingChangesAlert: FC<ModalPendingChangesAlertProps> = ({
   isChanged,
+  showMigrateOption = false,
 }) => {
   const { t } = useKubevirtTranslation();
-  const modalMsg = isChanged
-    ? t('The changes you have made require this VirtualMachine to be restarted.')
-    : t(
-        'Changes to the following settings will need to restart the VirtualMachine in order for them to be applied',
-      );
+
+  let modalMsg = t('The changes you have made require this VirtualMachine to be restarted.');
+  if (showMigrateOption) {
+    modalMsg = t('Restart or live migrate the Virtual Machine to apply the following changes:');
+  } else {
+    modalMsg = t('Restart the Virtual Machine to apply the following changes:');
+  }
+
```

Comment:
please try to avoid using let is it seems not really needed here. if I'm wrong please let me know :)

Reply:
I have removed this due to design changes, so I'm setting this conversation as resolved.